app/routes/people/person/positions/position.js

```diff
@@ -0,0 +1,27 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class PeoplePersonPositionsPositionRoute extends Route {
+  @service store;
+
+  async model() {
+    let person = this.modelFor('people.person');
+
+    let { positionId: mandatoryId } = this.paramsFor(
+      'people.person.positions.position'
+    );
+
+    let mandatory = this.store.findRecord('mandatory', mandatoryId, {
+      reload: true,
+      include: [
+        'mandate.role-board',
```

Comment:
The `classification`, `contacts` and `type-half` relationships are still lazy loaded, so I would include those here as well.

![image](https://user-images.githubusercontent.com/3533236/124262265-1e695780-db32-11eb-8e5e-a427ccf97f19.png)

Reply:
Hmm, contacts is still being fetched async. I think it might be a bug in mu-cl-resources. It's not returning the "empty" relationship so Ember doesn't know it's empty and fetches it async again.

Reply:
TODO 😬

Reply:
Yea, not much we can do about it I think. I'll investigate some more and create an issue where needed.